tests/register_test.js

```diff
@@ -1,9 +1,12 @@
 Feature('register');
 
 const NEW_USER = {
-    firstName: "Oleg",
-    lastName: "9",
-    email: Date.now() + '@test.com',
+    firstName: "Ivan",
+    lastName: "37",
+    emailField: 'ihappynavi' + Date.now() + '@test.com',
+    telephoneField: '380732761986',
```

Comment:
remove "Field" at the end, it is not required for object option

Reply:
done